src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -1021,5 +1016,78 @@ public void DisableExportingAudioData()
 
             _audioFrameDecodedCallback = null;
         }
+
+        /// <summary>
+        /// Enables to decode a video data for every frame.
+        /// </summary>
+        /// <remarks><para>The player must be in the <see cref="PlayerState.Idle"/> state.
+        /// And, <see cref="Multimedia.Display"/> must not be set.</para>
+        /// <para>A <see cref="VideoFrameDecoded"/> event is called in a separate thread(not in the main loop).</para>
+        /// <para>The video frame can be retrieved using a <see cref="VideoFrameDecoded"/> event as a media packet.
+        /// So if you change the media packet in the <see cref="VideoFrameDecoded"/> event, it will be displayed on the device.
+        /// The callback function holds the same buffer that will be drawn on the display device.
+        /// and the <see cref="MediaPacket"/> is available until it's destroyed by <see cref="MediaPacket.Dispose()"/>.
+        /// The packet has to be destroyed as quickly as possible after rendering the data
+        /// and all the packets have to be destroyed before <see cref="Unprepare"/> is called.</para></remarks>
```

Comment:
I think you don't need to a conjunction in this sentence.

       /// All the packets have to be destroyed before <see cref="Unprepare"/> is called.</para></remarks>